engine/release-notes/index.md

```diff
@@ -22,6 +22,63 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.7
+2021-06-02
+
+### Client
+
+* Suppress warnings for deprecated cgroups [docker/cli#3099](https://github.com/docker/cli/pull/3099).
+* Prevent sending `SIGURG` signals to container on Linux and macOS. The Go runtime
+  (starting with Go 1.14) uses `SIGURG` signals internally as an interrupt to
+  support preemptable syscalls. In situations where the Docker CLI was attached
+  to a container, these interrupts were forwarded to the container. This fix
+  changes the Docker CLI to ignore `SIGURG` signals [docker/cli#3107](https://github.com/docker/cli/pull/3107),
+  [moby/moby#42421](https://github.com/moby/moby/pull/42421).
+
+### Builder
+
+* Update BuildKit to version v0.8.3-3-g244e8cde [moby/moby#42448](https://github.com/moby/moby/pull/42448):
+    * Transform relative mountpoints for exec mounts in the executor to work around
+      a breaking change in runc v1.0.0-rc94 and up. [moby/buildkit#2137](https://github.com/moby/buildkit/pull/2137).
+    * Add retry on image push 5xx errors. [moby/buildkit#2043](https://github.com/moby/buildkit/pull/2043).
+    * Fix build-cache not being invalidated when renaming a file that is copied using
+      a `COPY` command with a wildcard. Note that this change invalidates
+      existing build caches for copy commands that use a wildcard. [moby/buildkit#2018](https://github.com/moby/buildkit/pull/2018).
+    * Fix build-cache not being invalidated when using mounts [moby/buildkit#2076](https://github.com/moby/buildkit/pull/2076).
+* Fix build failures when `FROM` image is not cached when using legacy schema 1 images [moby/moby#42382](https://github.com/moby/moby/pull/42382).
+
+### Logging
+
+* Update the hcsshim SDK to make daemon logs on Windows less verbose [moby/moby#42292](https://github.com/moby/moby/pull/42292).
+
+### Rootless
+
+* Fix capabilities not being honoured when an image was built on a daemon with
```

Comment:
nit: I think we're using American English? :smile: 

```suggestion
* Fix capabilities not being honored when an image was built on a daemon with
```

Reply:
Thx! Fixed 👍